### PR TITLE
Add ParseContext behaviour to trimming of well/group names

### PR DESCRIPTION
**Pull request by joakim-hove**
See comment in code.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=155 please

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=155 please

**Comment by joakim-hove** [comment]
jenkins build this opm-test=155 please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/Parser/ParseContext.hpp [hunk 1]
```diff
@@ -271,6 +271,14 @@ namespace Opm {
          */
         const static std::string PARSE_MISSING_SECTIONS;
 
+        /*
+          When defining wells and groups with the WELSPECS and GRUPTREE keywords
+          we do not allow leading or trailing spaces. The code in Schedule.cpp
+          will *unconditionally* remove the spaces, but with PARSSE_WGNAME_SPACE
```
Comment (atgeirr) [inline opm/parser/eclipse/Parser/ParseContext.hpp]: Typo: PARSSE_...

Comment (joakim-hove) [reply opm/parser/eclipse/Parser/ParseContext.hpp]: Fixed

**Pull request merged**